app/assets/javascripts/services/miq_service.js

```diff
@@ -124,11 +124,14 @@ ManageIQ.angular.app.service('miqService', ['$timeout', '$document', '$q', '$log
 
     return serializedObj;
   };
+
   this.handleFailure = function(e) {
     miqSparkleOff();
+
     if (e.message) {
-      $log.log(e.message);
+      console.error(e.message);
     }
```

Comment:
Looks like `$log` doesn't seem to work together with out toast-errors-in-devel-mode code :(.

And since we're not using it anywhere else, I see no reason to fix that...